addons/web/static/src/legacy/scss/fontawesome_overridden.scss

```diff
@@ -42,41 +42,45 @@
     }
 }
 
+// Define a list variable to store all the classes that need transformation
+$rtl-fa-classes: (
+    'fa-align-right',
+    'fa-align-left',
+    'fa-chevron-right',
+    'fa-chevron-left',
+    'fa-arrow-right',
+    'fa-arrow-left',
+    'fa-hand-o-right',
+    'fa-hand-o-left',
+    'fa-arrow-circle-right',
+    'fa-arrow-circle-left',
+    'fa-caret-right',
+    'fa-caret-left',
+    'fa-rotate-right',
+    'fa-rotate-left',
+    'fa-angle-double-right',
+    'fa-angle-double-left',
+    'fa-angle-right',
+    'fa-angle-left',
+    'fa-quote-right',
+    'fa-quote-left',
+    'fa-chevron-circle-right',
+    'fa-chevron-circle-left',
+    'fa-long-arrow-right',
+    'fa-long-arrow-left',
+    'fa-toggle-right',
+    'fa-toggle-left',
+    'fa-caret-square-o-right',
+    'fa-arrow-circle-o-left',
+    'fa-arrow-circle-o-right',
+    'fa-caret-square-o-left'
+);
+
 // This is rtl language specific fix
 // It will override the font awesome symbols and flip them 180 degree
-.o_rtl {
-    .fa {
-        &.fa-align-right,
-        &.fa-align-left,
-        &.fa-chevron-right,
-        &.fa-chevron-left,
-        &.fa-arrow-right,
-        &.fa-arrow-left,
-        &.fa-hand-o-right,
-        &.fa-hand-o-left,
-        &.fa-arrow-circle-right,
-        &.fa-arrow-circle-left,
-        &.fa-caret-right,
-        &.fa-caret-left,
-        &.fa-rotate-right,
-        &.fa-rotate-left,
-        &.fa-angle-double-right,
-        &.fa-angle-double-left,
-        &.fa-angle-right,
-        &.fa-angle-left,
-        &.fa-quote-right,
-        &.fa-quote-left,
-        &.fa-chevron-circle-right,
-        &.fa-chevron-circle-left,
-        &.fa-long-arrow-right,
-        &.fa-long-arrow-left,
-        &.fa-toggle-right,
-        &.fa-toggle-left,
-        &.fa-caret-square-o-right,
-        &.fa-arrow-circle-o-left,
-        &.fa-arrow-circle-o-right,
-        &.fa-caret-square-o-left {
-            transform: rotate(180deg);
-        }
+
+@each $class in $rtl-fa-classes {
+    .o_rtl .fa.#{$class} {
```

Comment:
This creates as many CSS rules as there are classes, to fix. There are several ways but I probably prefer extending a placeholder selector.

Reply:
@qsm-odoo I didn't know this feature. Good to know. It's fixed and indeed it's better like this.